src/metax_api/tests/api/rpc/base/views/file_rpc.py

```diff
@@ -0,0 +1,83 @@
+# This file is part of the Metax API service
+#
+# Copyright 2017-2018 Ministry of Education and Culture, Finland
+#
+# :author: CSC - IT Center for Science Ltd., Espoo Finland <servicedesk@csc.fi>
+# :license: MIT
+
+from django.core.management import call_command
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from metax_api.models import CatalogRecord, Directory, File
+from metax_api.tests.utils import TestClassUtils, test_data_file_path
+
+
+class FileRPCTests(APITestCase, TestClassUtils):
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Loaded only once for test cases inside this class.
+        """
+        super().setUpClass()
+
+    def setUp(self):
+        """
+        Reloaded for every test case
+        """
+        super().setUp()
+        call_command('loaddata', test_data_file_path, verbosity=0)
+        self._use_http_authorization()
+
+class DeleteProjectTests(FileRPCTests):
+
+    """
+    Tests cover user authentication, wrong type of requests and
+    correct result for successful operations.
+    """
+
+    def test_wrong_parameters(self):
+        #correct user, no project identifier
```

Comment:
May be nitpicking, but prefer one of the following format of single-line comments, since these have been widely used so far:

`# my comment`
`# My comment`

The only reason is to have the codebase look uniform (although that train already went when we let two commenting styles live...)